src/service/repo/signedMessageRepo.ts

```diff
@@ -18,7 +18,12 @@ const baseUrl = {
 
 interface SignedMessageRepoTypes {
   checkIsTargetAllowingAnonDM: (targetUserId: string) => Promise<any>;
-  sendSignedMessage: (channelId: string, message: string, channelType: number) => Promise<any>;
+  sendSignedMessage: (
+    channelId: string,
+    message: string,
+    channelType: number,
+    attachments: any
+  ) => Promise<any>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendSignedMessage` function now includes an `attachments` parameter. However, the type of this parameter is defined as `any`. It's better to avoid using `any` as it can lead to potential runtime errors. Instead, define a more specific type or interface for attachments.

```diff
-    attachments: any
+    attachments: AttachmentType[]
```
Where `AttachmentType` could be an interface that describes the structure of an attachment object.



<!-- This is an auto-generated comment by OSS CodeRabbit -->